app/models/katello/concerns/smart_proxy_extensions.rb

```diff
@@ -118,11 +118,25 @@ def container_gateway_users
         ::User.where(login: usernames['users'])
       end
 
+      def rhsm_url
+        # TODO: We shouldn't relate this to the Pulp feature and also expose it
```

Comment:
https://github.com/ekohl/smart_proxy_rhsm would do this. However, given there's a `content_source` I think we should also consider a single `smart_proxy_content` module. That will depend on if we ever want to allow splitting the concern.